styles/_variables.scss

```diff
@@ -1,7 +1,7 @@
 $font--primary: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI',
     Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji';
 
-$breakpoints: ('medium' '(min-width: 600px)', 'large' '(min-width: 1161px)');
+$breakpoints: ('medium' '(min-width: 1020px)', 'large' '(min-width: 1275px)');
```

Comment:
These breakpoints look quite close together - would you like to add a 'small' breakpoint so you could split the topic section cards into a two wide column?
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/18164832/172431854-f5c4d528-e1b6-4d2a-bcd9-a9945f85457f.png)

</details>

Reply:
Nice suggestion. I've added that.

Reply:
Nice suggestion. I've added that.